src/sheet/textbox-item.c

```diff
@@ -139,8 +139,8 @@ static void textbox_item_finalize (GObject *object)
 
 	item = TEXTBOX_ITEM (object);
 
```

Comment:
Should we maybe add an assert here?

Reply:
g_free(NULL) should not crash. So, I do not understand the reason why we should check or add assertions...

Reply:
See https://developer.gnome.org/glib/stable/glib-Memory-Allocation.html#g-free

Reply:
Yep, you are right.